.github/workflows/ci.yml

```
@@ -40,10 +40,10 @@ jobs:
           max_timeout: 1200
       - name: link checker
         run: |
-          `go env GOPATH`/bin/muffet ${URL} --buffer-size 8192  --exclude https://twitter.com --max-connections-per-host=8 --exclude 127.0.0.1 --exclude 'https://github.com/spiffe/spiffe/blob/main/standards/X509-SVID.md.*' --rate-limit 50 --timeout 60
+          `go env GOPATH`/bin/muffet ${URL} --buffer-size 8192  --exclude https://twitter.com --max-connections-per-host=8 --exclude 127.0.0.1 --exclude 'https://github.com/kumahq/kuma/pull' --exclude 'https://github.com/spiffe/spiffe/blob/main/standards/X509-SVID.md.*' --rate-limit 50 --timeout 60
```

[<code>\`...\`</code> as command substitution is legacy syntax](https://mywiki.wooledge.org/BashFAQ/082)
```suggestion
          $(go env GOPATH)/bin/muffet ${URL} --buffer-size 8192  --exclude https://twitter.com --max-connections-per-host=8 --exclude 127.0.0.1 --exclude 'https://github.com/kumahq/kuma/pull' --exclude 'https://github.com/spiffe/spiffe/blob/main/standards/X509-SVID.md.*' --rate-limit 50 --timeout 60
```